src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerCreateUserWithFileUploadTest.java

```diff
@@ -160,15 +154,9 @@ private CaseWorkerFileCreationResponse createCaseWorkerExpectedErrorDetails() {
             CaseWorkerConstants.MISSING_REGION).build());
         errors.add(JsrFileErrors.builder().rowId("9").filedInError("regionId").errorDescription(
             CaseWorkerConstants.MISSING_REGION).build());
-        errors.add(JsrFileErrors.builder().rowId("10").filedInError(ROLE_FIELD).errorDescription(
```

Comment:
Currently, we have issues in the file where for the some of the rows, the Secondary Location Id is not evaluated as formula. So removed those rows from the file.

Reply:
👍